src/layouts/default.vue

```diff
@@ -60,8 +56,9 @@ const embeddedPage = {
   setup() {
     const mainContentRef = ref(null)
     const mainRef = ref(null)
-    const { store } = useContext()
-    const isReferredFromCc = store.state[NAV].isReferredFromCc
+
+    const navStore = useNav()
+    const isReferredFromCc = computed(() => navStore.isReferredFromCc)
```

Comment:
Is this computed even necessary? It's just wrapping a ref in another ref as far as I can tell. Can `isReferredFromCc` be used directly off the store?

Reply:
Yes, `navStore.isReferredFromCc` is not reactive unless you either put it inside `computed` or use `storeToRefs`.